Add Arraylake comparison page to docs (#2066)
earth-mover/icechunk: icechunk-python/docs/docs/arraylake.md

---
title: Arraylake
---

# Arraylake

[Icechunk](https://icechunk.io) is a free, open-source (Apache 2.0) transactional storage engine for Zarr.
[Arraylake](https://docs.earthmover.io) is Earthmover's managed cloud platform, built on top of Icechunk.
Both are made by [Earthmover](https://earthmover.io/).

Icechunk is a fully capable, standalone product -- you never need Arraylake to use it.

> Arraylake adds operational, collaboration, and data delivery features on top of Icechunk for teams that want a managed experience.

Reviewer: ```suggestion
Arraylake adds operational, collaboration, and data-delivery features on top of Icechunk for teams that want a managed experience.
```

| [OPeNDAP / DAP2](https://docs.earthmover.io/flux/dap2) | :material-minus: | :material-check: |

### Operations & Monitoring

Arraylake automates routine maintenance and gives visibility into repository health.

| Feature | Icechunk | Arraylake |
|---|---|---|
| [Garbage collection & data expiration](https://docs.earthmover.io/guide/04-garbage-collection) | :material-check: You run it | :material-check: Scheduled, runs on managed compute |
| Monitoring & metrics dashboards | :material-minus: | :material-check: Repo-level and org-level |
| [Webhooks & Slack notifications](https://docs.earthmover.io/guide/07-notifications) | :material-minus: | :material-check: Commit events |
| [Performance tuning](https://docs.earthmover.io/guide/05-performance) | :material-check: Manual configuration | :material-check: `arraylake repo tune` benchmarking |

### Support & Pricing

| Feature | Icechunk | Arraylake |
|---|---|---|
| Pricing | Free forever (Apache 2.0) | Free tier (read-only) + [Professional tier](https://docs.earthmover.io/pricing) |
| Support | Community (GitHub, Slack) | Priority support |

## When to Use Which

> **Use Icechunk on its own** if you are comfortable managing your own cloud infrastructure, don't need a web UI or access control beyond cloud IAM, and want full control with zero cost and zero vendor dependency.

Reviewer: ```suggestion
**Use Icechunk on its own** if you are comfortable managing your own cloud infrastructure, don't need a web UI or access control beyond cloud IAM, and want full control with no additional cost or vendor dependency.
```

> **Use Arraylake** if you need team collaboration with role-based access, want a web UI for managing repositories, need to serve data via standard protocols (OGC, OPeNDAP), or want managed operations like garbage collection, credential vending, and monitoring.

Reviewer 2: Or if you want high quality professional premade datasets f rom teh maraketplace

## No Lock-in

> Arraylake stores your data in Icechunk format in your own object storage (bring your own bucket), following the open Icechunk Format Specification.

Reviewer: ```suggestion
Arraylake stores your data in Icechunk format in your own object storage (bring your own bucket), following the open Icechunk Format Specification.
If you discontinue your Arraylake subscription, you can still read and write all of your data using Icechunk.
```
